lib/fastlane/actions/verify_build.rb

```diff
@@ -67,7 +67,9 @@ def self.update_with_profile_info(app_path, values)
         values['provisioning_uuid'] = plist['UUID']
         values['team_name'] = plist['TeamName']
 
-        raise "Inconsistent identifier found" unless plist['Entitlements']['application-identifier'] == "#{values['team_identifier']}.#{values['bundle_identifier']}"
+        split_bundle_identifier = plist['Entitlements']['application-identifier'].partition('.').last
```

Comment:
I don't mean to be pedantic, but it would be great if you could switch that up to use `split` as it is a more commonly used API and has slightly more predictable and slightly different behavior from `partition`.